source/agora/node/Ledger.d

```diff
@@ -227,10 +227,15 @@ public class Ledger
 
         this.enroll_man.clearExpiredValidators(block.header.height);
 
-        foreach (enrollment; block.header.enrollments)
-            if (!this.enroll_man.addValidator(enrollment,
+        foreach (idx, ref enrollment; block.header.enrollments)
+            if (auto r = this.enroll_man.addValidator(enrollment,
                 block.header.height, this.utxo_set.getUTXOFinder()))
+            {
+                log.fatal("Error while adding a new validator: {}", r);
+                log.fatal("Enrollement #{}: {}", idx, enrollment);
```

Comment:
Can you fix up the typo `Enrollement` in a future PR?

Reply:
Also